src/core_plugins/console/public/src/sense_editor/mode/input.js

```diff
@@ -1,27 +1,24 @@
-let acequire = require('acequire');
-require('ace');
-require('ace/mode-json');
+import ace from 'ace';
```

Comment:
We're no longer requiring `ace/mode-json`, is that intentional?

Reply:
We don't seem to need to.  The syntax checking works without it.